pkg/virtctl/ssh/loadTest.go

```diff
@@ -0,0 +1,136 @@
+package ssh
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/pkg/virtctl/templates"
+)
+
+func NewLoadTestCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
```

Comment:
I'm still working on this, it is what I use for testing performance impact etc.
One thing I still want/need to do is add a random stress test mode that keeps adding and closing tunnels for a longer time to see that we really don't leave anything behind.
But from my tests so far, the results look really good and I'll attach a grafana dump later.